Files.Launcher/MessageHandlers/FileOperationsHandler.cs

```diff
@@ -660,6 +661,42 @@ await Win32API.SendMessageAsync(connection, new ValueSet()
                         { "PickedObject", pickedObject }
                     }, message.Get("RequestID", (string)null));
                     break;
+
+                case "ReadCompatOptions":
+                    {
+                        var filePath = (string)message["filepath"];
+                        var compatOptions = Extensions.IgnoreExceptions(() =>
+                        {
+                            using var compatKey = Registry.CurrentUser.OpenSubKey(@"SOFTWARE\Microsoft\Windows NT\CurrentVersion\AppCompatFlags\Layers");
```

Comment:
@BanCrash regarding the "run as admin" option not working could you check in regedit HKCU what value is set in the above key for an .exe where you have set "run as admin" through Files or explorer?

Reply:
I have to check later with debugger, because now without debugger both options are working. In any case, here you have:

Files: 
![image](https://user-images.githubusercontent.com/11770760/140758821-adb62e8e-04e3-464b-9b13-6718573b6dbe.png)

Windows explorer:
![image](https://user-images.githubusercontent.com/11770760/140759135-c3c2fa27-6c84-4361-b63f-e40c220094f5.png)

I'm using the Debug version of Files - Dev, but executing directly from Start menu instead using Visual Studio.